.circleci/config.yml

```diff
@@ -24,40 +26,72 @@ jobs:
-                name: Publish @pxblue/cra-template-routing-authentication-typescript
-                command:   |
-                  cd authentication-typescript && yarn publish:package -b $CIRCLE_BRANCH
+      docker:
+        - image: circleci/node:12.9.1-browsers
+      steps:
+        - checkout
+        - run:
+            name: Authenticate with registry
+            command: |
+              echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > ~/.npmrc
+              echo "//registry.yarnpkg.com/:_authToken=$NPM_TOKEN" >> ~/.npmrc
+        - run:
+            name: Publish @pxblue/cra-template-blank
+            command:  |
+              cd blank && yarn publish:package -b $CIRCLE_BRANCH
+        - run:
+            name: Publish @pxblue/cra-template-blank-typescript
+            command:   |
+              cd blank-typescript && yarn publish:package -b $CIRCLE_BRANCH
+        - run:
+            name: Publish @pxblue/cra-template-routing
+            command:  |
+              cd routing && yarn publish:package -b $CIRCLE_BRANCH
+        - run:
+            name: Publish @pxblue/cra-template-routing-typescript
+            command:   |
+              cd routing-typescript && yarn publish:package -b $CIRCLE_BRANCH
+        - run:
+            name: Publish @pxblue/cra-template-authentication
+            command:  |
+              cd authentication && yarn publish:package -b $CIRCLE_BRANCH
+        - run:
+            name: Publish @pxblue/cra-template-routing-authentication-typescript
+            command:   |
+              cd authentication-typescript && yarn publish:package -b $CIRCLE_BRANCH
+
+    tag:
+      docker:
+        - image: circleci/node:12.9.1-browsers
+      steps:
+        - checkout
+        - gh/setup
+        - run:
+            name: Tag @pxblue/cra-template-blank
+            command:  |
+              cd blank && yarn tag:package -b $CIRCLE_BRANCH -s -cra-template-blank
+        - run:
+            name: Tag @pxblue/cra-template-blank-typescript
+            command:   |
+              cd blank-typescript && yarn tag:package -b $CIRCLE_BRANCH -s -cra-template-blank-template
+        - run:
+            name: Tag @pxblue/cra-template-routing
+            command:  |
+              cd routing && yarn tag:package -b $CIRCLE_BRANCH -s -cra-template-routing
+        - run:
+            name: Tag @pxblue/cra-template-routing-typescript
+            command:   |
+              cd routing-typescript && yarn tag:package -b $CIRCLE_BRANCH -s -cra-template-routing-typescript
+        - run:
+            name: Tag @pxblue/cra-template-authentication
+            command:  |
+              cd authentication && yarn tag:package -b $CIRCLE_BRANCH -s -cra-template-authentication
+        - run:
+            name: Tag @pxblue/cra-template-routing-authentication-typescript
+            command:   |
+              cd authentication-typescript && yarn tag:package -b $CIRCLE_BRANCH -s -cra-template-routing-authentication-typescript
```

Comment:
```suggestion
        - run:
            name: Tag @pxblue/cra-template-authentication-typescript
            command:   |
              cd authentication-typescript && yarn tag:package -b $CIRCLE_BRANCH -s -cra-template-authentication-typescript
```

Reply:
or you can pull out the suffixes while you're in here.

Reply:
I've removed the suffixes.